database/models/BoxCrime.js

```diff
@@ -0,0 +1,20 @@
+var mongoose = require('mongoose');
+
+
+var boxCrimeSchema = new mongoose.Schema({
```

Comment:
Nope. With all the rebasing it just crept back in somehow. I'm updating everything now to reflect both of my PR's along with the PR Tyler submitted last night :)